config/jobs/kubernetes-sigs/sig-windows/release-master-windows.yaml

```diff
@@ -117,6 +117,9 @@ periodics:
         env:
           - name: IMAGE_VERSION
             value: "127.1.20230417" # pin the Windows nodes to a specific OS patch version while investigating an issue with container updates
+          # skip Procmount security context test until https://github.com/kubernetes/kubernetes/issues/126180 is resolved (then remove GINKGO_SKIP env var entirely)
+          - name: GINKGO_SKIP
+            value: \[LinuxOnly\]|device.plugin.for.Windows|RebootHost|\[sig-autoscaling\].\[Feature:HPA\]|ignore.ProcMount.Specific.SecurityContext
```

Comment:
Should we skip the proc mount job on the Presubmit too? I guess we won't be able to validate the fix if we do that but we will have failing job on all PRs that modify windows

Reply:
Yea, i thought we should leave it to validate the fix